components/esp_http_server/include/esp_http_server.h

```diff
@@ -608,6 +608,9 @@ typedef enum {
      */
     HTTPD_411_LENGTH_REQUIRED,
 
+    /* Incoming payload is too large */
+    HTTPD_413_PAYLOAD_TOO_LARGE,
```

Comment:
Suggest changing to:

`HTTPD_413_CONTENT_TOO_LARGE`

same comment at other places too

Reply:
Happy to change this by all means, but all the other names appear to follow the literal HTTP status code names as defined in spec, so just curious as to the motivation?

Reply:
I think by other places, he meant at other changes in this PR. Anyways, we have updated the commit to have that change. The MR is in internal review right now. The commit will be reflected on GitHub once the MR gets merged internally. Thanks for your contribution.